components/sections/supershardians/ShardiansList.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Container, Flex, HStack, SimpleGrid } from "@chakra-ui/react";
+import Card from "components/common/ProfileInfoCard";
+
+type ShardiansProps = {
+  categories: Array<any>;
+  filteredData: Array<any>;
+  changeCategory: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+const Shardians = ({ categories, filteredData, changeCategory }: ShardiansProps) => {
+  return (
+    <Flex bg="brand.grey-5">
+      <Container pl={0} pr={0} maxW="container.xl" pt={[6, 8, 12]} pb={[6, 8, 12]}>
+        <HStack pb={12} pl={[3, 5, 0]} pr={[3, 5, 0]}>
+          {categories.map((category, index) => {
+            return (
+              <Button
```

Comment:
On click of the filter button - this focus state gets activated which looks wierd.  
![image](https://user-images.githubusercontent.com/7558499/168416345-923f2281-8d8c-4f6b-a982-26e847a41d30.png)

Reply:
Addressed